cpp-ch/local-engine/Disks/ObjectStorages/MetadataStorageFromRocksDB.h

```diff
@@ -22,6 +22,7 @@
 #include <Disks/ObjectStorages/IMetadataStorage.h>
 #include <Disks/ObjectStorages/MetadataOperationsHolder.h>
 #include <rocksdb/db.h>
+#include <shared_mutex>
```

Comment:
useless header

Reply:
It is needed otherwise building would fail